misc/userscripts/qute-pass

```diff
@@ -108,6 +117,10 @@ def dmenu(items, invocation, encoding):
 
 
 def main(arguments):
+    if not arguments.url:
+        argument_parser.print_help()
+        return ExitCodes.FAILURE
```

Comment:
Instead of doing this yourself, you should be able to do `nargs=1` instead of `nargs='?'` above.